lienflash/lienflash-fe
# Sn feature/job form

#### What’s this PR do?
- Adds a components folder for the components which won't be connecting to Redux
- Creates a two page form for user to add new jobs
- Passes unit tests for form components
- Installs package for tooltip and adds tooltip functionality to form field
- Adds styling and media queries for the form which can be used globally when we setup our SASS files
#### Where should the reviewer start?
- Review JobForm, JobFormStepOne,  and JobFormStepTwo files

#### How should this be manually tested?
- In browser, fill out the form. You shouldn't be able to click 'next' if you're missing a required field. 
- When you hover over the 'What is this?' text you should see a message appear.
- In terminal run `npm test`
#### Any background context you want to provide?
- I'd like to change the 'What is this?' text to an information or question mark icon if we have time to play around with styling.
- Page two doesn't have any required fields per conversation with client (it didn't sound like these needed to be required). We can easily make other fields required based on feedback we get from client the next time we check-in.
#### What are the relevant tickets?
resolves #28 
resolves #20 
resolves #14 
#### Screenshots (if appropriate)
This is the first section of the form and you can see the tooltip hover feature: 
![Screen Shot 2020-10-26 at 5 07 53 PM (2)](https://user-images.githubusercontent.com/62770843/97239349-024a3500-17b1-11eb-97f5-4eb2d7e48b0a.png)

#### Questions:
- The code for the form is pretty ugly because of all the attributes I've used. If you have any ideas for how to clean up the code that would be appreciated. 
- Do you think we should add any more restrictions for the input fields? 
#### Need Help/Support
- Need to ask client what text he would like on the hover feature to explain what the _"Date of Substantial Completion is"_
